OloEngine/tests/Lua/LuaBindingTest.cpp

```
@@ -0,0 +1,1987 @@
+#include "OloEnginePCH.h"
+#include <gtest/gtest.h>
+
+#define SOL_ALL_SAFETIES_ON 1
```

Clang (14.0.6)

[warning] macro 'SOL_ALL_SAFETIES_ON' used to declare a constant; consider using a 'constexpr' constant

(cppcoreguidelines-macro-usage)